pyiron/atomistics/structure/atoms.py

```diff
@@ -192,6 +193,9 @@ def __init__(
             self.dimension = len(self.positions[0])
         else:
             self.dimension = 0
+        self.plot3d = MethodType(Visualize.plot3d, self)
```

Comment:
I think it's a bit weird to define the `Visualize` object and then never instantiate it and instead pick the methods like this.  I'd prefer simple functions in the `_visualize` module, but @samwaseda  and @liamhuber discussed this at length already, so I won't block this and follow what they agreed on.

Reply:
Yes, I also find this problematic. I think the core of the issue here is that `Visualize` is just `object`-type (which Marvin rightly points out no longer needs to be explicit), but then its public methods assume that the `self` argument should be of type `Atoms`. The original formulation of this PR (which I fussed at...) really instantiated the `Visualization` class and passed it an `Atoms` instance as data.

This smells like a messed up inheritance tree to me. The closest thing I could see would be having something like `_AtomsBase(ASEAtoms)`, `_AtomsVisualization(AtomsBase)`, `_AtomsAnalysis(AtomsBase)`, etc., then finally making our public-facing class incorporate all the functionality using multiple inheritance: `Atoms(_AtomsVisualization, _AtomsAnalysis)`. 

However, if we chose to go this multiple inheritance route I think it's something that could be built *on top of* this PR, rather than stopping this PR to do it that way.